test/unit/tcframe/spec/io/GridIOSegmentManipulatorTests.cpp

```diff
@@ -75,4 +75,14 @@ TEST_F(GridIOSegmentManipulatorTests, Printing_Failed_ColumnsMismatch) {
     }
 }
 
+TEST_F(GridIOSegmentManipulatorTests, Clearing_Segment_Successful) {
```

Comment:
The convention is that the test must have `Parsing_` prefix because it is related to parsing, and grouped with all other parsing tests. Please fix the naming